storage-aws-s3/src/main/java/org/rutebanken/helper/aws/repository/S3BlobStoreRepository.java

```diff
@@ -49,10 +51,14 @@ public S3BlobStoreRepository(S3Client s3Client) {
 
     @Override
     public InputStream getBlob(String objectName) {
-        return s3Client.getObject(
-                GetObjectRequest.builder().bucket(containerName).key(objectName).build(),
-                ResponseTransformer.toInputStream()
-        );
+        try {
+            return s3Client.getObject(
+                    GetObjectRequest.builder().bucket(containerName).key(objectName).build(),
+                    ResponseTransformer.toInputStream()
+            );
+        } catch (NoSuchKeyException e) {
+            return null;
+        }
```

Comment:
Looks fine to me 👍 This is a bit of a wart in AWS S3 client - using exceptions for flow control, that is - so handling it like this is fine.

Of course returning an `Optional<InputStream>` could express the semantics of the return value better, but the current implementation and the other changes in the PR are less intrusive and won't cause issues downstream, so I'm fine with this.